code/game/gamemodes/cult/cult.dm

```diff
@@ -37,10 +37,10 @@
 	false_report_weight = 10
 	restricted_jobs = list("Chaplain","AI", "Cyborg", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Head of Personnel")
 	protected_jobs = list()
-	required_players = 24
-	required_enemies = 4
```

Comment:
Let's not nerf  cult team members either.

Reply:
So recommended enemies is the actual antags or?... It's so terribly vague to me